### DISCUSSION TITLE
带有"-"字符的邮箱地址无法通过邮箱检测

### DISCUSSION THREAD

**Discussion by zhullyb** [walinejs/waline#585] [Q&A] [answered]
有访客反馈邮箱中带有"-"字符，过不到waline的邮箱检测。
![](https://user-images.githubusercontent.com/43305366/135625446-3cf171ed-721c-457f-8e56-6aa1b5a190ef.png)
有去翻过源码，可惜我连vue的基础都没有，请求帮助。
https://github.com/walinejs/waline/blob/d5c58cc1a6a091c025c33f462277d03411cc393b/packages/client/src/components/CommentBox.vue#L405

**Comment by Mister-Hope** [comment]
收到，我这边后续优化一下

**Comment by Mister-Hope** [comment, marked as answer]
邮箱检测应该是已经去除了的。虽然我也没什么印象啥时候把它删掉了.....

**Reply by lizheming** [reply]
我处理的

**Reply by Mister-Hope** [reply]
hhhhh 好歹改个正则啊，因为有的人真的会随手填错邮箱，把它加回去比较好。
